build_resolvers/test/resolver_test.dart

```diff
@@ -195,4 +197,36 @@ void main() {
       }, resolvers: AnalyzerResolvers());
     });
   });
+
+  group('The ${_isFlutter ? 'flutter' : 'dart'} sdk', () {
+    test('can${_isFlutter ? '' : ' not'} resolve types from dart:ui', () async {
+      return resolveSources({
+        'a|lib/a.dart': '''
+              import 'dart:ui';
+
+              class MyClass {
+                final Color color;
+
+                MyClass(this.color);
+              } ''',
+      }, (resolver) async {
+        var entry = await resolver.libraryFor(AssetId('a', 'lib/a.dart'));
+        var classDefinition = entry.getType('MyClass');
+        var color = classDefinition.getField('color');
+
+        if (_isFlutter) {
+          expect(color.type.name, equals('Color'));
```

Comment:
[nit] I think we typically haven't used `equals` in this repo

Reply:
:man_shrugging: not worth rerunning the ci imo